modules/core/src/lib/layer.ts

```diff
@@ -184,6 +184,14 @@ export type UpdateParameters<LayerT extends Layer> = {
   changeFlags: ChangeFlags;
 };
 
+export type DrawOptions = {
+  context: LayerContext;
+  moduleParameters: any;
```

Comment:
Not new in this PR, but the name can be really confusing. I think it's better called moduleSettings or shaderInputProps?